### DISCUSSION TITLE
PC maps?

### DISCUSSION THREAD

**Discussion by Developer2704** [viciious/d32xr#102] [Ideas]
Would it be possible to make a version that uses the full 1mb ram in the Sega CD32X to run "PC-ish maps? If the SNES can run dimilar maps with 128kb of RAM and a 16-bit SuperFX II chip at 20mhz the 32x should probably be able to do it. Sadly I am not experienced in 32x coding or anything involving SH-2s. Would it even be possible to use the PSX maps for Thy Flesh Consumed?

**Comment by viciious** [comment]
The 32X only has direct access to 256KiB of RAM of its own RAM, everything else is practically external storage. So no, CD32X without the cartridge would not be able to run even Jaguar Doom PC Doom maps, let alone PC Doom maps.

**Reply by Developer2704** [reply]
Thanks for the feedback! How did they get it to run on the SNES? I bet the Saturn could atleast be fixed up to run PSX DooM and not run at 7fps. Hopefully someone will do that someday..... Or Hiddenpalace will discover the fabled hardware accelerated version that supposedly ran at 35fps. By the way, how much Everdrive ram does d32xr actually use? And the typo-riddled, inconsistent Sega of America docs suggest you can use the SCD ram but I highly doubt them

**Reply by viciious** [reply]
Remember the SNES port didn't have any floors or ceilings + ran at super low resolution + the AI was super simplified.

To answer your second question D32XR doesn't use any extra RAM to avoid vendor locking itself to any particular model or revision of a flashcard or emulator.

You can use the SegaCD RAM on the MD, but not on the 32X part of the combo.

**Comment by Developer2704** [comment]
![Screenshot_20240511_123333_Microsoft 365 (Office)](https://github.com/viciious/d32xr/assets/111246017/906da1ab-6c14-4a7f-9121-9fae40fbd07a)

**Reply by Developer2704** [reply]
Sorry to be annoying but in a demo could the ram be _theoretically_ used via very, very, very, very slow access through the MD?

**Reply by viciious** [reply]
Theoretically - yes, but only theoretically. On top of that, about 0.5MiB of the extra RAM would be used for sound effects alone.

**Comment by Developer2704** [comment]
Sorry to post on this old discussion, but I have heard of DooM 32x Delta which supposedly has the PC maps. Is this legit or does it use the Everdrive RAM. There isn't any way to run the PC levels on 256kb of RAM (to my knowlege) unless they somehow found out how to use the MCD ram.

**Reply by viciious** [reply]
There are plenty of videos and screenshots of how Doom 32X Delta looks. Have you done any investigating on your own before posting here?
